apps/judicial-system/web/src/routes/Court/RestrictionCase/HearingArrangements/HearingArrangements.tsx

```diff
@@ -154,15 +161,32 @@ export const HearingArrangements = () => {
         title={formatMessage(titles.court.restrictionCases.hearingArrangements)}
       />
       <FormContentContainer>
-        {workingCase.comments && (
-          <Box marginBottom={5}>
-            <AlertMessage
-              type="warning"
-              title={formatMessage(m.comments.title)}
-              message={workingCase.comments}
-            />
-          </Box>
-        )}
+        <Box marginBottom={5}>
+          <AlertMessage
```

Comment:
Can we extract this as a specific Alert component and share it between investigation and restriction case `HearingArrangements` components so we can avoid maintaining the same component in two places?

Reply:
Yes we can.